relational/table.py

```diff
@@ -318,22 +318,33 @@ def _gen_units(self, row, proj):
             row[self.index(self.NET_NUM_UNITS_BMR_DATA)] = source
         else:
             dbi_net = _get_dbi_units(proj)
+            ppts_net = proj.field('market_rate_units_net', PPTS.NAME)
```

Comment:
Hmm unrelated, but I thought we had said we wanted to use the total project units here and not just the market rate units?

Reply:
There's issue #48: iiuc this is just a naming issue in the schema-less

Reply:
sg